src/components/annual-report/ReportPrintHeader.js

```diff
@@ -0,0 +1,64 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useConfigurations } from '../../utils/index.js'
+import { CORE_GROUP_CODE } from '../report-parameter-selector/index.js'
+import styles from './ReportPrintHeader.module.css'
+
+/** Shown only in print to indicate report parameters */
+export const ReportPrintHeader = ({ reportParameters }) => {
+    const configurations = useConfigurations()
+
+    const groupName = React.useMemo(() => {
+        if (!reportParameters?.groupID) {
+            return null
+        }
+        if (reportParameters.groupID === CORE_GROUP_CODE) {
+            return i18n.t('[Core]')
+        }
+        const group = configurations.groups.find(
+            (group) => group.code === reportParameters.groupID
+        )
+        return group.name
+    }, [configurations, reportParameters])
+
+    if (!reportParameters || Object.keys(reportParameters).length === 0) {
+        return null
+    }
+
+    const { orgUnitName, orgUnitLevelName, periods } = reportParameters
+
+    return (
+        <div className={styles.showOnlyInPrint}>
+            <h1 className={styles.title}>
```

Comment:
Semantically, I guess this makes sense, but feels a bit strange that our h1 tag is generally hidden?

Reply:
Heh, yeah maybe... it's the only one in the app though, and it makes sense in print, so it seems good enough to me though 🙂